.github/workflows/regression-tests.yml

```diff
@@ -90,7 +90,8 @@ jobs:
       # Run tests that don't need a simulator.
     - name: Install Python testing dependencies
       run: |
-        pip install nox
+        conda install --yes sqlite=3.39
```

Comment:
Is this specific version necessary? If so, add a comment as to why.

Reply:
Just previous version used. But it is still failing. I have no idea.